python/changelog.py

```diff
@@ -0,0 +1,108 @@
+import argparse
+import json
+import os
+import sys
+
+from git import Repo
+
+
+def changelog(arguments: list = sys.argv[2:]):
```

Comment:
Can we re-use https://github.com/taichi-dev/taichi/blob/master/python/taichi/make_changelog.py ?

Reply:
done